docs/clis/diagcfg/metric/disable.md

```diff
@@ -42,7 +42,7 @@ diagcfg metric disable -d dataDir [flags]
 
 ```
 
-  -d, --dataDir string   Data directory
+  -d, --dataDir string   Data directory (default "/Users/ryan/node/data")
```

Comment:
the default is /Users/ryan ?

Reply:
I hope not! There must be something in the script that writes that. I'm not sure where, nor why. We don't have this default path displayed for most commands, so seems odd to include it anywhere.